cozy-theme/nav.html

```diff
@@ -15,7 +15,8 @@
 
           {%- block site_name %}
             <a class="navbar-brand" href="https://docs.cozy.io/">{{ config.site_name }}</a>
-            <a class="navbar-brand" href="https://docs.cozy.io/en">en</a>&nbsp;<a class="navbar-brand" href="https://docs.cozy.io/fr/index_fr">fr</a>
+            <a class="navbar-brand" href="https://docs.cozy.io/en">en</a>
+            <a class="navbar-brand" href="https://docs.cozy.io/fr/index_fr">fr</a>
```

Comment:
Why did you remove the blank space?

Reply:
I don't say, here, to not remove it, I say: could you explain yourself in the Pull Request comments?

Reply:
I don't know, it seems to be unnecessary and unused. I think @clochix could know if it's important or not.